code/game/gamemodes/gamemode.dm

```diff
@@ -650,3 +650,15 @@ proc/get_nt_opposed()
 			if (L.stat != DEAD) //They're alive!
 				GLOB.living_crew |= M
 
+//proc returns the amount of alive active humans onboard Ishimura (space turfs are excluded), the rest are considered marooned
+/datum/game_mode/proc/get_alive_crewmates_count()
+	var/crew_count = 0
+	for(var/datum/mind/M in GLOB.living_crew)
+		var/mob/living/L = M.current
+		if(!L.client || L.client.is_afk(2 MINUTES))	//activity check
```

Comment:
I'm going to leave this comment for posterity, but 2 minutes might be low. Only way we'll know is by testing.

I'm saying this because sometimes humans have to hide, be silent, and don't move as to not make any sound.

Reply:
Well, I figured that 1 minute will indeed be too low, however two minutes is enough, since it's quite a lot of time in terms of movement or doing anything in game.

Reply:
Fair enough. Thanks for your work!